client/src/pages/Students/editables/PlansPane.js

```diff
@@ -149,10 +149,10 @@ const PlansTableItem = ({ plan: { _id, name, numYears, startYear }, programme })
                         color={programme ? c.lightBlue : c.red}
                         rounded="10px"
                         onClick={(e) => {
-                            e.stopPropagation();
-                            window.open(`/programmes/${programme._id}`, "_blank");
+                            programme && e.stopPropagation();
+                            programme && window.open(`/programmes/${programme._id}`, "_blank");
                         }}
```

Comment:
Why not chain all of them into one line xd

programme && e.stopPropogation() && window.open(`/programmes/${programme._id}`, "_blank");

Reply:
chaining doesnt seem to work :(

Reply:
rip